File: mcp/src/minecraft/femtocraft/api/ISuctionPipe.java
package femtocraft.api;

import net.minecraftforge.fluids.IFluidHandler;

/**
 * 
 * @author Itszuvalex
 *
 */
public interface ISuctionPipe extends IFluidHandler {
	
	/**
	 * 
	 * @return Pressure of this pipe.  Liquids will flow from high pressure to low pressure pipes.
	 */
	public float getPressure();
}

File: src/main/scala/com/itszuvalex/femtocraft/api/transport/ISuctionPipe.scala
package com.itszuvalex.femtocraft.api.transport

import net.minecraftforge.fluids.IFluidHandler

/**
 * @author Itszuvalex
 */
trait ISuctionPipe extends IFluidHandler {
  /**
   * @return Pressure of this pipe. Liquids will flow from high pressure to low pressure pipes.
   */
  def getPressure: Int
}
